Exercise_1.py

```diff
@@ -1,20 +1,27 @@
+##Exercise_1 : Implement Stack using Array.
 class myStack:
-     def __init__(self):
-         
-     def isEmpty(self):
-         
-     def push(self, item):
-         
-     def pop(self):
-        
-        
-     def peek(self):
-        
-     def size(self):
-         
-     def show(self):
-         
+    def __init__(self):
+        self.L=[]
+        self.size=0
 
+    def isEmpty(self):
+        return self.L == []
+
+    def push(self,item):
+        self.L.append(item)
+        self.size=self.size+1
+
+    def pop(self):
```

Comment:
Shouldn't the size variable be decreased when an element is popped?

Reply:
Yes. I was in hurry to submit. Will recheck from now on.